storage/shard.go

```diff
@@ -168,6 +167,12 @@ type dbShardEntry struct {
 	series         series.DatabaseSeries
 	index          uint64
 	curReadWriters int32
+	reverseIndex   struct {
```

Comment:
same comment about the name, field name is weird since we just refer to the index itself as an index

Reply:
I wanted to use `index` but the entry already uses that for the `UniqueIndex` int64. What would you like it called instead?

Reply:
ah....in that case maybe keep calling this `reverseIndex` and rename the other one `uniqueIndex` so its not ambiguous. Although now that you've pointed that out, I'm not sure if I want you to rename the reverseIndex field in the namespace anymore

Reply:
lol yeah this pretty much the thought process that led to me to name both of them `reverseIndex`. Will leave as is unless there's a better suggestion.